### PR TITLE
[ADD] l10n_in: added settings to enable India ENet Payment CSV Generator

### PR DESCRIPTION
**Pull request by khva-odoo**
Added a new setting to which activates India ENet Payment CSV Generator module.

taskid: 3825529
Related PR: https://github.com/odoo/enterprise/pull/62958

### CODE REVIEW

File: addons/l10n_in/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -29,6 +29,12 @@
                     </setting>
                 </block>
             </app>
+            <block id="print_vendor_checks_setting_container" position="inside">
+                <setting id="vendor_payment_order_file" company_dependent="1"
+                    help="Generate Vendor Payment Order file(csv file), upload to your bank to make the payments" invisible="country_code != 'IN'">
```
Comment (malb-odoo) [inline addons/l10n_in/views/res_config_settings_views.xml:34]: Since we add a help here, could you export the pot again :smile:

**Review by malb-odoo — COMMENTED** [review]
Almost good, thanks of the PR !

File: addons/l10n_in/views/res_config_settings_views.xml (hunk 1 above)
Comment (khva-odoo) [reply addons/l10n_in/views/res_config_settings_views.xml:34]: Updated the pot according to your suggestion

**Review by malb-odoo — APPROVED** [review]
LGTM, i will let the others review the enterprise pr, thanks for this 😄

**Review by rugo-odoo — APPROVED** [review]

**Comment by rugo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**